tools/include/eosio/abigen.hpp

```diff
@@ -64,21 +64,25 @@ namespace eosio { namespace cdt {
          ret.ricardian_contract = rcs[get_action_name(decl)];
 
          if (action_name.empty()) {
+            /*
```

Comment:
It doesn't seem appropriate to take out validation of names for everyone even if they are not using the new hashed name system.